index.php

```diff
@@ -42,10 +81,12 @@
 
                 <nav class="main-navigation">
                     <ul class="main-navigation__list">
+                        <?php foreach ($projects as $key): ?>
```

Comment:
Используй соответсвующее пространство имен. $key не очевидная переменная
```suggestion
                        <?php foreach ($projects as $project): ?>
```